### PR TITLE
move `freq` params to dataloader

### PR DESCRIPTION
**Pull request by you-n-g**
move `freq` params to dataloader

## Description
The frequency should be tranparent to data handler.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [X] Add new feature
- [ ] Update documentation

### CODE REVIEW

File: qlib/data/data.py [hunk 1]
```diff
@@ -961,8 +961,7 @@ def features(
         is a provider class.
         """
         disk_cache = C.default_disk_cache if disk_cache is None else disk_cache
-        if C.disable_disk_cache:
```
Comment (you-n-g) [inline qlib/data/data.py:964]: @zhupr Please help to check the correctness of the modification

File: qlib/data/dataset/handler.py [hunk 2]
```diff
@@ -71,38 +71,58 @@ def __init__(
...
 
         # Setup data loader
         assert data_loader is not None  # to make start_time end_time could have None default value
 
+        # what data source to load data
         self.data_loader = init_instance_by_config(
             data_loader,
             None if (isinstance(data_loader, dict) and "module_path" in data_loader) else data_loader_module,
             accept_types=DataLoader,
+            **kwargs,
```
Comment (bxdd) [inline qlib/data/dataset/handler.py:94]: Is it necessary to pass kwargs to `data_loader`?
I think all arguments of `data_loader` should be in config.

Comment (bxdd) [reply qlib/data/dataset/handler.py:94]: Is handlers in contrib should keep consistent with example? Pass arguments to data_loader with kwargs or config?

**Comment by you-n-g** [comment]
@bxdd  I've removed `kwargs` now.
Please let me know if there are any other problems  :) w

**Pull request merged**